schema.json

```diff
@@ -0,0 +1,2729 @@
+                        "widget": "wysiwygSingleLineText"
+                    },
+                    "additionalPackingInstructions": {
+                        "widget": "singleLineText"
+                    }
+                },
+                "order": [
+                    "productVariantId",
+                    "additionalPackingInstructions"
+                ]
+            }
+        },
+        "CollectionMetadata": {
+            "default": {
+                "properties": {
+                    "collectionId": {
+                        "widget": "singleLineText"
+                    },
+                    "externalCollectionTitle": {
+                        "widget": "singleLineText"
+                    }
+                },
+                "order": [
+                    "collectionId",
+                    "externalCollectionTitle"
+                ]
+            }
+        }
+    },
+    "services": {
+        "takeshapetestsite": {
+            "title": "takeshapetestsite",
+            "namespace": "Shopify",
+            "provider": "shopify",
+            "serviceType": "graphql",
+            "authenticationType": "oauth2Bearer",
+            "options": {
+                "shop": "takeshapetestsite",
+                "endpoint": ""
+            },
+            "id": "takeshapetestsite"
+        }
+    },
+    "indexedShapes": {
+        "Shopify_Product": {
+            "queries": {
+                "list": {
+                    "name": "Shopify_products",
+                    "ignoreFields": [
+                        "availablePublicationCount",
+                        "publicationCount",
+                        "publishedOnCurrentPublication",
+                        "productPublications",
+                        "publications",
+                        "resourcePublications",
+                        "resourcePublicationsV2",
+                        "unpublishedChannels",
+                        "unpublishedPublications",
+                        "metadata"
+                    ],
+                    "objectDepthLimit": 1
+                },
+                "get": {
+                    "name": "Shopify_product",
+                    "ignoreFields": [
+                        "availablePublicationCount",
+                        "publicationCount",
+                        "publishedOnCurrentPublication",
+                        "productPublications",
+                        "publications",
+                        "resourcePublications",
+                        "resourcePublicationsV2",
+                        "unpublishedChannels",
+                        "unpublishedPublications",
+                        "metadata"
+                    ],
+                    "objectDepthLimit": 1
+                }
+            },
+            "triggers": [
+                {
+                    "type": "schedule",
+                    "query": "list",
+                    "interval": 5
```

Comment:
Every 5 minutes is aggressive since we have webhooks. I'd bump this up to `1440` (24 hours). The user can configure a lower interval if they want.